html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <div>
+        <label for="email">Email</label> <br />
+        <input id="email" type="email"> <br />
+      </div>
+      <div>
+        <label for="iscoding">Why is coding cool?</label> <br />
```

Comment:
Did you test clicking on the label to put focus on the form control it is labeling? This one won't work as intended.

Reply:
![html-forms gif2](https://user-images.githubusercontent.com/113414352/206002731-4f4e6eb5-ede9-47d2-b39c-eaeebd9e2e13.gif)